tests/unit/cn/kblock_builder_test.c

```diff
@@ -192,6 +193,18 @@ test_setup(struct mtf_test_info *lcl_ti)
     return 0;
 }
 
+int
+test_setup2(struct mtf_test_info *lcl_ti)
+{
+    mapi_inject(mapi_idx_wbti_create, 0);
+    mapi_inject(mapi_idx_wbti_destroy, 0);
+    mapi_inject(mapi_idx_wbti_next, 0);
+    mapi_inject(mapi_idx_wbti_prefix, 0);
+    mapi_inject(mapi_idx_wbt_read_kmd_vref, 0);
+
+    return 0;
+}
```

Comment:
I wonder how this works when three of these funcs are defined as void!

Reply:
No idea but did the job for me :-)